### ISSUE TITLE
Can not create a file 

### ISSUE THREAD

**Issue by tokenizr** [ipfs/go-mfs#74]
Hi all. I am trying to create file in already created folder using `mfs`.
```
err = mfs.Mkdir(nd.FilesRoot, "outbox", mfs.MkdirOpts{
		Flush:     true,
		Mkparents: true,
	})
	if err != nil {
		fmt.Errorf("Eror when trying to create mfs  directory: %v\n", err)
		return err
	}

	outboxLookup, err := mfs.Lookup(nd.FilesRoot, "outbox")
	outbox, _ := outboxLookup.(*mfs.Directory)

         // here I need to pass a format.Node
         // but I can not understood what I need to pass there if the file is new
         // and when passing nil I am getting an error 
	err = outbox.AddChild("seed", nil)

	if err != nil {
		fmt.Printf("Error caused: %v\n ", err)
		return err
	}
```

Please help me to solve this error!

**Comment by aschmahmann** [comment]
This repository is no longer maintained and has been copied over to [Boxo](https://github.com/ipfs/boxo).  In an effort to avoid noise and crippling in the Boxo repo from the weight of issues of the past, we are closing most issues and PRs in this repo.  Please feel free to open a new issue in Boxo (and reference this issue) if resolving this issue is still critical for unblocking or improving your usecase.

You can learn more in the [FAQs for the Boxo repo copying/consolidation effort](https://github.com/ipfs/boxo/wiki/Copied-or-Migrated-Repos-FAQ).
